standards/genc/src/main/java/org/codice/countrycode/standards/genc/package-info.java

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
❓ What does this file do?

Reply:
With the jaxb upgrade, this was needed to define the GENC XML schema; without it the unmarshaling was failing.